temba/channels/migrations/0129_auto_20211103_1310.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 2.2.24 on 2021-11-03 13:10
```

Comment:
@ihor-palii - quick question: this migration will also update the existing msgs on the database?

Reply:
No, it will set the segments count to NULL

Reply:
Got it, so related to the existing messages is there a way that we can fill the segments count? Like a manual migration that we can run in the background?

Reply:
We have saved the responses from the channel logs. But it's not JSON data. I'll try somehow get the number of segments from those logs.

Reply:
Got it. The response is like this:
```
HTTP/1.1 201 Created
Content-Length: 922
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since
Access-Control-Allow-Methods: GET, POST, DELETE, OPTIONS
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: ETag
Connection: keep-alive
Content-Type: application/json
Date: Mon, 22 Nov 2021 04:25:04 GMT
Strict-Transport-Security: max-age=31536000
Twilio-Concurrent-Requests: 1
Twilio-Request-Duration: 0.134
Twilio-Request-Id: xxxx
X-Api-Domain: api.twilio.com
X-Home-Region: us1
X-Powered-By: AT-5000
X-Shenanigans: none

{"sid": "xxxxxx", "date_created": "Mon, 22 Nov 2021 04:25:04 +0000", "date_updated": "Mon, 22 Nov 2021 04:25:04 +0000", "date_sent": null, "account_sid": "xxxxx", "to": "+xxxxx", "from": "+xxxxx", "messaging_service_sid": null, "body": "This number is not set up to receive calls. We do receive texts and will respond ASAP. For other inquiries, call the Director's Action Line at xxxxxx.", "status": "queued", "num_segments": "1", "num_media": "0", "direction": "outbound-api", "api_version": "2010-04-01", "price": null, "price_unit": "USD", "error_code": null, "error_message": null, "uri": "/2010-04-01/Accounts/xxxxxx/Messages/xxxxxx.json", "subresource_uris": {"media": "/2010-04-01/Accounts/xxxxx/Messages/xxxx/Media.json"}}
```
So the idea would be using regex to identify a JSON object and create an instance of it, then you will be able to handle it as an object to grab the number of segments.